### ISSUE TITLE
Diagrams not working on Apple Silicon

### ISSUE THREAD

**Issue by goens** [lf-lang/vscode-lingua-franca#27]
Hi,

just tried out the new plugin on my new Macbook with Apple silicon, it's really good how it 'just works'!

Unfortunately the diagrams seem to be broken. There is no diagram shown and the options tab writes "no options provided by the diagram server". 

For context: 
```
java --version
openjdk 17.0.2 2022-01-18
OpenJDK Runtime Environment Homebrew (build 17.0.2+0)
OpenJDK 64-Bit Server VM Homebrew (build 17.0.2+0, mixed mode, sharing)

uname -a
Darwin AG-MacBook-Pro.lan 21.2.0 Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:41 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T6000 arm64

code --version
1.64.1
d6ee99e4c045a6716e5c653d7da8e9ae6f5a8b03
arm64
```

**Comment by Soroosh129** [comment]
For me, the diagram feature sometimes works and sometimes doesn't, depending on the program.

Going through a few [C tests](https://github.com/lf-lang/lingua-franca/tree/master/test/C), here are two examples where one works and the other doesn't:

AfterCycles.lf
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/4221770/162847062-07c76068-6a47-427c-aac8-ebf65c966358.png">

After.lf
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/4221770/162847107-242fcb05-ec6c-491c-a06f-fa779c31c924.png">

**Comment by NiklasRentzCAU** [comment]
The Issues with Mac have been identified and will be fixed in the next KLighD release. There are two small changes that you have to apply to your code as well to make the diagrams work again:
- when starting the languge server Java process, you have to include a `-Djava.awt.headless=true` command line parameter
- In your `org.lflang.diagram.synthesis package you will have to add the line `KlighdDataManager.getInstance().registerBlacklistedProperty(NamedInstanceUtil.LINKED_INSTANCE);`
This will only work once you have updated to the next KLighD Release. We will notify you or create PRs once this can be finalized.

Thank you for the reports and your patience for us fixing this!

**Comment by petervdonovan** [comment]
This issue is also assumed resolved, like #52, for the same reason. Please reopen if not.
